test/tests/functional/pbs_hook_execjob_end.py

```diff
@@ -257,3 +257,74 @@ def test_execjob_end_delete_request(self):
                           interval=2)
             msg = "Got expected log_msg on host:%s" % host
             self.logger.info(msg)
+
+    def test_execjob_end_reject_request(self):
+        """
+        Test to make sure hook job reject message should appear in mom log
+        in case sister mom went down before executing execjob_end hook
+        """
+
+        if len(self.moms) != 2:
+            self.skip_test(reason="need 2 mom hosts: -p moms=<m1>:<m2>")
+        self.momA = self.moms.values()[0]
+        self.momB = self.moms.values()[1]
+
+        # Create hook
+        hook_name = "execjob_end_logmsg7"
+        self.server.create_import_hook(hook_name, self.attr, self.hook_body)
+
+        # Submit a multi-node job
+        a = {'Resource_List.select':
+             '1:ncpus=1:host=%s+1:ncpus=1:host=%s' %
+             (self.momA.shortname, self.momB.shortname),
+             'Resource_List.place': 'excl'}
```

Comment:
why do you need to specify the host in the job request and instead do something like -lselect=2:ncpus=1 -lplace=scatter:excl

Reply:
I take this comment back because if you change it to < 2 then you do need to specify the host you would want your job to run on.